ERCS/erc-7579.md

```diff
@@ -268,6 +274,8 @@ interface IModule {
 }
 ```
 
+Note: A single module that is of multiple types MAY decide to pass `moduleTypeId` inside `data` to `onInstall` and/or `onUninstall` methods, so those methods are able to properly handle installation/uninstallation for various types.
```

Comment:
does this need to be in the standard? this seems like module specific behaviour that would be encoded offchain like installation/uninstallation data

Reply:
I don't think it SHOULD be here, but it MAY be here :)
I just found it to be useful for people implementing modules according to the standard. More of best practice, which, btw, is not only about encoding data, but about decoding it in the onInstall/onUninstall as well